src/main/java/org/opentestsystem/ap/ims/service/ItemService.java

```diff
@@ -35,9 +35,9 @@
 @Component
 public class ItemService {
 
-    private static final boolean NO_DO_NOT_COMMIT_ON_SAVE = false;
+    private static final boolean KEEP_CHANGES_PRIVATE = false;
```

Comment:
Yes.  I have latched on to more business concepts instead of technical Git stuff.  For example "isItemFileOnMaster"  is now "isItemPublic".  IMS is fairly straightforward.  The common lib has the meat of what we are doing.  It started to get a little confusing but I think I 'm starting to zero in on a good feel for how the calls flow, particularly in ItemRepository and GitClient.  Definitely would like input on that stuff.  Could it be more functional for example?  Could it be more fluent?  It is probably easiest to pull it up in an IDE and click through the code.  Thanks for all your input.